packages/mdc-toolbar/README.md

```diff
@@ -136,6 +290,98 @@ The provided modifiers are:
 
 | Class                                | Description                             |
 | -------------------------------------| --------------------------------------- |
-| `mdc-toolbar--fixed`                 | Make toolbar fixed to top of screen.    |
+| `mdc-toolbar--fixed`                 | Makes toolbar fixed on top and have persistent elavation |
+| `mdc-toolbar--waterfall`             | Removes fixed toolbar persistent elavation and makes it to gain elevation when a user begins to scroll down the page |
+| `mdc-toolbar--fixed-lastrow-only`    | Makes only last row of fixed toolbar anchored on top |
+| `mdc-toolbar--flexible`              | Makes toolbar first row has flexible space |
 | `mdc-toolbar__section--align-start`  | Makes section aligns to the start.      |
 | `mdc-toolbar__section--align-end`    | Makes section aligns to the end.        |
+
+
+## JS Usage
+
+### Including in code
+
+#### ES2015
+
+```javascript
+import {MDCToolbar, MDCToolbarFoundation} from 'mdc-toolbar';
```

Comment:
I think you are right...But I wonder why do we have all modules documented wrong in the past...